cpp/src/io/parquet/experimental/hybrid_scan.cpp

```diff
@@ -97,9 +97,6 @@ std::vector<cudf::size_type> hybrid_scan_reader::filter_row_groups_with_dictiona
   parquet_reader_options const& options,
   rmm::cuda_stream_view stream) const
 {
-  CUDF_EXPECTS(row_group_indices.size() == dictionary_page_data.size(),
```

Comment:
Removed as this is an erroneous check: as dictionary_page_data.size() == number of eligible dictionary columns x `row_group_indices.front().size()`